app/src/main/res/values/strings.xml

```diff
@@ -42,13 +42,23 @@
 <!-- AI ASSISTANT -->
     <string name="ask_assistant">Ask the AI assistant!</string>
     <string name="prompt">Prompt</string>
-    <string name="go">Go</string>
+    <string name="ask">Ask</string>
     <string name="settings">Settings</string>
     <string name="settings_subtitle">Remember to click on the \'go\' button after changing the settings</string>
     <string name="provide_final_activities">Provide final activities with date and time (recommended for trips shorter than a week)</string>
     <string name="use_user_interests">Use only your profile\'s interests</string>
     <string name="use_participants_interests">Use all participants\'s interests</string>
     <string name="close">Close</string>
+    <string name="assistant_initial_screen">Please enter a prompt to receive activity recommendations. You can mention what types of activities you would like to do on your trip, or leave it empty for general recommendations based on the trip details. Fee free to use the settings button to customize the recommendations.</string>
```

Comment:
Also in the string "Remember to click ..." the button is now ask not go

Reply:
Done!